pages/appointmentDetails/[id].js

```diff
@@ -0,0 +1,74 @@
+import EventCard from "@/components/Event";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import Details from "../../components/AppointmentDetails";
+export default function EventDetailPage() {
+  const router = useRouter();
+  const { id } = router.query;
+  const { data, isLoading } = useSWR(id ? `/api/appointments/${id}` : null);
+
+  async function handleSubmitNotes(event) {
+    event.preventDefault();
+    const notes = event.target.elements.notes.value;
+    try {
+      const response = await fetch(`/api/appointments/${data._id}`, {
+        method: "PATCH",
+        body: JSON.stringify(notes),
+        headers: { "Content-type": "application/json" },
+      });
+      if (response.ok) {
+        router.reload();
+      }
+    } catch (error) {
+      console.error(error);
+    }
+  }
+
+  async function handleSubmitForm(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    try {
+      const response = await fetch(`/api/uploadDocument/${data._id}`, {
+        method: "PATCH",
+        body: formData,
+      });
+      if (response.ok) {
+        const result = await response.json();
+        router.reload();
```

Comment:
This is the equivalent to pressing the refresh button in your browser. Is that the intended behaviour?

If not, I would use the mutate function you can destructure from the useSWR hook

Reply:
Done